.github/ISSUE_TEMPLATE/docs-request.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Doc request
+about: Request a new topic to help us improve
+
+---
+
+**Help us make content visible**
+
+- Tell us what you search terms you used, and how you searched docs.
+- Tell us what docs you found that didn't address your concern.
+
+**Describe the new topic**
+
+- Tell us where this topic should go in the Table of Contents.
+- Write an abstract. In one **short** paragraph, describe what this topic will cover.
+- Create an outline for the new topic. We'll help review the outline and approve that before anyone writes a topic.
+
+In addition, tell us why this topic is needed.
```

Comment:
I'd move this to first, before "Tell us where this topic should go." And perhaps "Explain why this topic is important."

Reply:
I also re-wrote the next bullet items to match voice and person.